eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -219,6 +233,20 @@ public final void execute() throws MojoFailureException {
                     ),
                     ex
                 );
+            } catch (final TimeoutException ex) {
+                throw new MojoExecutionException(
+                    String.format(
```

Comment:
@volodya-lombrozo you can use `Logger.format("%[ms]s", timeout)`, see https://log.jcabi.com/decors.html

Reply:
@yegor256 done.